testutils/testutils.go

```diff
@@ -13,3 +16,54 @@ func NewTestLogger() (*zap.Logger, error) {
 
 	return loggerConfig.Build()
 }
+
+func NewTestSugaredLogger() (*zap.SugaredLogger, error) {
+	logger, err := NewTestLogger()
+	if err != nil {
+		return nil, err
+	}
+	return logger.Sugar(), nil
+}
+
+func NewSampleNATSStatefulSetUnStruct(opts ...SampleOption) *unstructured.Unstructured {
```

Comment:
What does `SampleNATS` mean? why can't it be just `NATS`?

Reply:
updated